### ISSUE TITLE
关于论文中posetrack方面的一个小问题

### ISSUE THREAD

**Issue by Ananileaf** [microsoft/human-pose-estimation.pytorch#156]
论文4.2小节中提到

> It contains 514 videos including 66,374 frames in total, split into 300, 50 and 208 videos for training, validation and test set respectively. 

请问你们使用的是posetrack2017的数据集吗，而不是2018的，求解答。
